### PR TITLE
Enhancement pagination for export page

### PR DESCRIPTION
**Pull request by RignonNoel**
| Q                                                      | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                      | Enhancement
| Tickets (_issues_) concerned               | [TV-163](https://fjnr-inc.atlassian.net/browse/TV-163)

---

## Personnal note

I've found a bug on the API during the integration on the Frontend, i created an issue to fix it.

https://fjnr-inc.atlassian.net/browse/TV-193

## What have you changed ?
Add pagination on export admin page

## How did you change it ?
 - Refactor the page to use a list of export in place of hardcoded/copy-pasted HTML/TS
 - Use new version of export call in API

## Screenshots
Screenshot_2019-05-23 Thèsez-vous
Screenshot_2019-05-23 Thèsez-vous(1)

## Verification :
 -  [x] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [x] This Pull-Request fully meets the requirements defined in the issue
 -  [x] I added or modified the attached tests

### CODE REVIEW

File: src/app/components/pages/admin/export/export.component.html [hunk 1]
```diff
@@ -1,90 +1,38 @@
...
-      </div>
-    </div>
-
-    <div class="export-page__list__item" (click)="exportOrderLine()">
-      <i class="export-page__list__item__icon fas fa-file-invoice-dollar"></i>
-      <div class="export-page__list__item__name">
-        {{ 'export.orderlines' | translate }}
-      </div>
+    <div class="export-page__list__title" (click)="exportUser()">
+      Type d'export
```
Comment (sarahJeanneT) [inline src/app/components/pages/admin/export/export.component.html]: Types d'export

File: src/app/components/pages/admin/export/export.component.html [hunk 2]
```diff
@@ -1,90 +1,38 @@
...
     </div>
 
-    <div class="export-page__list__item" (click)="exportPackage()">
-      <i class="export-page__list__item__icon fas fa-hand-holding-usd"></i>
+    <div class="export-page__list__item" *ngFor="let type of exports" (click)="export(type.code)">
+      <i class="export-page__list__item__icon {{type.icon}}"></i>
       <div class="export-page__list__item__name">
-        {{ 'export.packages' | translate }}
+        {{ type.name | translate }}
```
Comment (sarahJeanneT) [inline src/app/components/pages/admin/export/export.component.html]: c'est ici que tu listes les types d'exports? c'est drôle comme on assume que "espace de travail" est au singulier alors que les autres sont au pluriel xD

File: src/app/components/pages/admin/export/export.component.html [hunk 3]
```diff
@@ -1,90 +1,38 @@
 <div class="export-page">
-  <h1 class="export-page__title nt-title">
-    {{ 'export.title' | translate }}
```
[the remark this answers is not in the document]
Comment (RignonNoel) [reply src/app/components/pages/admin/export/export.component.html]: Done, j'ai mis une condition

File: src/app/components/pages/admin/export/export.component.html (hunk 2 above)
Comment (RignonNoel) [reply src/app/components/pages/admin/export/export.component.html]: Bien vu, j'ai changé dans la liste TS

**Pull request merged**